src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java

```diff
@@ -421,50 +428,50 @@ public void testDiagramsAndGraphics() throws Exception {
         assertTrue(TestUtils.getRequestsDone(1, server).contains(String.format("/v1/substations?networkUuid=%s&variantId=%s&substationId=BBE1AA&substationId=BBE2AA", NETWORK_UUID_STRING, VARIANT_ID)));
 
         //get the lines map data of a network
-        getNetworkElementsInfos(studyNameUserIdUuid, rootNodeUuid, "LINE", "MAP", List.of(), "[]");
-        getNetworkElementsInfos(studyNameUserIdUuid, rootNodeUuid, "LINE", "MAP", List.of("BBE1AA"), "[]");
+        getNetworkElementsInfos1(studyNameUserIdUuid, rootNodeUuid, "MAP", mapper.writeValueAsString(createRequestBody("LINE", List.of())), "[]");
```

Comment:
optional : use null for all mapper.writeValueAsString(createRequestBody("LINE", List.of())). 

but it' not bloking